.github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md

```diff
@@ -0,0 +1,189 @@
+---
+name: Calamari XCM testing and onboarding checklist.
+about: Perform cross chain transfer tests in order to open HRMP channel with Calamari!
+
+---
+
+# Calamari XCM Testing And Onboarding Checklist
+
+## Process Overview:
+
+- [ ] Test XCM between Calamari and your runtime locally with polkadot-launch.
+
+- [ ] Become a parachain on Rococo.
+
+- [ ] Calculate and fund your parachain's sovereign account on Rococo.
+
+- [ ] Open HRMP channels with Calamari.
+
+- [ ] Assets registrations.
+
+- [ ] Complete XCM tests between parachains.
+
+- [ ] Next steps discussion.
+
+* Manta Team Contact:
+    - Georgi (XCM Eng): @Ghz (Telegram)
+    - Shumo (Co-founder, Tech.): @xstec (Telegram)
+
+## Local XCM Integration
+
+- As a first step we insist that both teams first run all tests on a local network.
+- For that you can download the latest manta binary from the Releases page.
+- Then use polkadot-launch to launch a `calamari-local` or `calamari-dev` network for testing.
```

Comment:
Perhaps we could provide a template of config file.

Reply:
Yeah i'll just put it at the end and link to it.